### PR TITLE
Update for Union.pm

### PR DESCRIPTION
**Pull request by Andre-J**
See https://github.com/finance-quote/finance-quote/issues/226 for discussion.

### CODE REVIEW

File: lib/Finance/Quote/Union.pm [hunk 1]
```diff
@@ -159,6 +151,6 @@ exchange, name, date, price, last.
 
 =head1 SEE ALSO
 
-UNION (Union Invest), http://www.union-invest.de/
+UNION (Union Investment), http://www.union-investment.de/
```
Comment (fellen) [inline lib/Finance/Quote/Union.pm:154]: Still no https:?

File: lib/Finance/Quote/Union.pm [hunk 2]
```diff
@@ -109,41 +113,29 @@ sub unionfunds
...
+Finance::Quote::Union	- Obtain quotes from UNION (Union Investment).
 
 =head1 SYNOPSIS
 
     use Finance::Quote;
 
     $q = Finance::Quote->new;
 
-    %stockinfo = $q->fetch("unionfunds","975788");
+    %stockinfo = $q->fetch("unionfunds","DE0008491002");
```
Comment (fellen) [inline lib/Finance/Quote/Union.pm:138]: Perhaps mention somewhere the change from german WKNs to ISIN?

**Comment by bpschuck** [comment]
You're not being ignored. I had to go out of state for a personal issue. It will be at least a few days before I have time to review and merge the PR.
Not to be a pest, but also look at the files Changes and Modules-README.yml.  Making the appropriate changes there will be helpful. 
Thank you.

File: lib/Finance/Quote/Union.pm (hunk 2 above)
Comment (Andre-J) [reply lib/Finance/Quote/Union.pm:141]: > Perhaps mention somewhere the change from german WKNs to ISIN?

You are right, I added that and the https:

**Comment by bpschuck** [comment]
I finally have some free cycles. I merged some recent changes to the F::Q repo to your fork while getting ready to merge the PR.
Thanks!

**Pull request merged**